services/user/Events/include/services/user/Events.hpp

```diff
@@ -62,6 +62,12 @@ namespace UserService
                     psibase::MethodNumber  event,
                     std::uint8_t           column);
 
+      // Same as `addIndex` except that multiple columns can be specified at the same time.
```

Comment:
I'm not sure how useful this really is. There's still a lot of boilerplate using it. There's also no performance benefit to batching here, so I'd prefer to keep the service interface minimal.

I think a better approach would be to create a wrapper using reflection:
```
auto indexes = EventIndexCreator<Tokens>{}.history();
indexes.created().addIndices("tokenId", "creator");
```

Reply:
I like this idea